google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -31,6 +31,9 @@ appengine.flex.dockerfile.location.label=Dockerfile path\:
 appengine.flex.config.destination.folder.label=Choose destination folder\:
 appengine.flex.config.choose.destination.folder.window.title=Choose destination folder
 appengine.flex.config.destination.chooser.title=Choose Generated Configuration Destination Folder
+appengine.flex.config.destination.create.dir=Do you want to create the non-existing directory below?
+appengine.flex.config.destination.overwrite=The file below already exists. Do you want to overwrite it?
```

Comment:
I wouldn't use a UI positional relative word like 'below'.  It would be easy to move the UI around and not realize this needed to change.